navigation-material/src/androidTest/java/com/google/accompanist/navigation.material/BottomSheetNavigatorTest.kt

```diff
@@ -127,7 +126,10 @@ internal class BottomSheetNavigatorTest {
                 secondDestinationCompositions++
                 onDispose { secondDestinationCompositions = 0 }
             }
-            Box(Modifier.size(64.dp).testTag(secondDestinationContentTag))
+            Box(
```

Comment:
We can probably don't need to change the formatting here :)

Reply:
This happened automatically from Command-option-L to clean up the imports and code style. I can change it back.